src/Security/InMemoryUserEmailAddressRepository.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Security;
+
+use CultuurNet\UDB3\Model\ValueObject\Web\EmailAddress;
+
+final class InMemoryUserEmailAddressRepository implements UserEmailAddressRepository
+{
+    private static array $mappedUserIds = [];
+
+    public static function addUserEmail(string $userId, EmailAddress $emailAddress)
+    {
+        self::$mappedUserIds[$userId] = $emailAddress;
+    }
+
+    public function getEmailForUserId(string $userId): ?EmailAddress
+    {
+        if (array_key_exists($userId, self::$mappedUserIds)) {
+            /** @var EmailAddress $emailAddress */
```

Comment:
You can add the following docblock on the property instead and then you don't need to add the `@var` annotation here I think, and then you can just do `return self::$mappedUserIds[$userId];`

```
/**
 * @var EmailAddress[] $mappedUserIds
 */
private static array $mappedUserIds = [];
```